.github/workflows/bindings-nodejs.yml

```diff
@@ -128,7 +128,7 @@ jobs:
 
       - name: Install Dependencies
         working-directory: sdk/src/client/bindings/nodejs/
-        run: yarn
+        run: yarn install --ignore-scripts
```

Comment:
Maybe it's my lack of yarn knowledge now, but do we even need the dependencies to check the format :trollface: ? Why ?

Reply:
The dependencies check format, it's like needing to installing rustfmt first, just that you don't have to do this manually with Rust :trollface:

Reply:
`yarn format-check` would fail if we don't install the dependencies ? Weird...

Reply:
`format-check` runs `prettier --ignore-path .eslintignore -c {,*/**/}*.{ts,js,json}` and prettier is a dependency which needs to be installed first

Reply:
Ok thanks for the info

Reply:
Could you install only that dependency?

Reply:
Yes, but seems like that's only possible with npm https://github.com/yarnpkg/yarn/issues/5201#issuecomment-511945863 and then we would have yarn and npm mixed, do we prefer that?